kubescripts/k8s-full-delete.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+microk8s kubectl delete --all services
+microk8s kubectl delete --all deployments
+microk8s kubectl delete --all pods
+microk8s kubectl delete --all jobs
+microk8s kubectl delete --all networkpolicy
+microk8s kubectl delete --all pvc
+microk8s kubectl delete --all secret
+microk8s kubectl delete --all ingress
+
+microk8s kubectl delete pv run-pv
+microk8s kubectl delete pv admin-webroot-pv
+microk8s kubectl delete pv main-webroot-pv
+microk8s kubectl delete pv nginx-conf-pv
+microk8s kubectl delete pv downloadables-pv
+microk8s kubectl delete pv docs-pv
+microk8s kubectl delete pv confd-cache-pv
+
+microk8s kubectl delete storageclass local-storage
```

Comment:
I don't understand k8s too well so just to make sure, this isn't going to delete the content of local directories mapped to volumes, right?

Reply:
No, this should not delete our local directories which are mapped to the volumes.
This command is specific to Kubernetes and is used to delete a StorageClass resource within a Kubernetes cluster.